packages/terra-worklist-data-grid/src/WorklistDataGrid.jsx

```diff
@@ -45,23 +48,47 @@ const propTypes = {
    */
   columnHeaderHeight: PropTypes.string,
   /**
-   * String that specifies the default height for rows in the grid. Any valid CSS width value is accepted.
-   * To override the default value, provide height for the row that needs to be overridden.
+   * String that specifies the height for the rows in the grid. Any valid CSS value is accepted.
    */
   rowHeight: PropTypes.string,
   /**
    * Number indicating the index of the column that represents row header. Index is 0 based and cannot exceed one less than the number of columns in the grid.
    */
   rowHeaderIndex: PropTypes.number,
+  /**
+   * Callback function that is called when a selectable cell is selected. Parameters: `onRowSelect(rowId, columnId)`.
```

Comment:
Can we follow the existing Terra convention for documenting function proptypes:

eg. `function(value)`

From terra-form-select-field: https://engineering.cerner.com/terra-ui/components/cerner-terra-core-docs/form-select/select-field
![image](https://github.com/cerner/terra-framework/assets/42594047/cfc6ff3b-7c57-476a-8b60-fc5fa8f7215c)

```suggestion
   * Callback function that is called when a selectable cell is selected. Parameters: `function(rowId, columnId)`.
```

Reply:
Done